SensusUI/ProtocolsPage.cs

```diff
@@ -152,35 +156,59 @@ public ProtocolsPage()
                         actions.Add("Ungroup");
                 }
 
+                bool isPriorToStart = !(DateTime.Now > new DateTime(selectedProtocol.StartDate.Year, selectedProtocol.StartDate.Month, selectedProtocol.StartDate.Day, selectedProtocol.StartTime.Hours, selectedProtocol.StartTime.Minutes, 0));
+                if (!selectedProtocol.ScheduledToStart && !selectedProtocol.Running && !selectedProtocol.StartImmediately && isPriorToStart)
+                {
+                    actions.Remove("Start");
+                    actions.Insert(0, "Schedule To Start");
```

Comment:
I'd prefer using Start to cover both cases. I don't think the latter is any clearer to the user, and the scheduled status of the protocol should be reflected in the list.